blog/2022-11-04-dont-create-your-own-i18n-library.md

```diff
@@ -0,0 +1,255 @@
+---
+slug: dont-create-your-own-i18n-library
+
+title: "Creating your custom i18n solution? Don't."
+
+authors: [jcizmar]
+
+image: "/img/blog/dont-custom-i18n.png"
+
+description: "Don't create you custom i18n solution. There is a lot of localization tooling. It's reinventing the
+wheel."
+
+tags: [i18n, localization, programming, typescript]
+---
+
+Sometime before, when we did a market analysis, we asked many companies how they solved localization in their project.
```

Comment:
sometime before+adverb 
I would delete sometime before and start the phrase just: When we...

Reply:
market analysis in Tolgee